helm/cas-provision/templates/linter/linterServiceAccount.yaml

```
@@ -0,0 +1,12 @@
+# Deploy only in the linter namespace
+{{ if eq .Release.Namespace .Values.linterNamespace }}
+
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: {{ .Release.Name }}-{{.Release.Namespace}}-circleci
```

No need to explicitly define that CircleCI is the tool doing the linting
```suggestion
metadata:
  name: {{ .Release.Name }}-linter
```